data_discovery_ai/model/keywordModel.py

```diff
@@ -294,7 +294,6 @@ def load_saved_model(trained_model: str) -> Optional[load_model]:
         saved_model = load_model(model_file_path, compile=False)
         return saved_model
     except Exception as e:
-        print(e)
         logger.info(
```

Comment:
should be error logging instead of info

Reply:
loggings in expect block have been changed to error type in the latest commit